.gitignore

```diff
@@ -12,3 +12,4 @@
 **/*.egg-info
 **/*.eslintcache
 **/*.swp
+**/yarn.lock
```

Comment:
is this ignore actually working? are there supposed to be yarn.lock files as a part of this PR?

Reply:
That line should be removed from the commit. Yes, it is fine for the yarn.lock file to included in the commit.